bundles/lpar/xenial-newton-stable.yaml

```diff
@@ -0,0 +1,349 @@
+    charm: cs:xenial/ceilometer-agent-2
+  ceph-mon:
+    charm: cs:xenial/ceph-mon-3
+    num_units: 3
+    options:
+      expected-osd-count: 5
+    to:
+    - lxc:1
+    - lxc:2
+    - lxc:3
+  ceph-osd:
+    charm: cs:xenial/ceph-osd-3
+    num_units: 5
+    options:
+      osd-devices: /mnt/ceph
+      osd-reformat: 'no'
+    to:
+    - '1'
+    - '2'
+    - '3'
+    - '4'
+    - '5'
+  #  Swift instead for this scenario
+  #ceph-radosgw:
+  #  charm: cs:xenial/ceph-radosgw-5
+  #  num_units: 1
+  #  options:
+  #    use-embedded-webserver: true
+  #  to:
+  #  - lxc:1
+  cinder:
+    charm: cs:xenial/cinder
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+      block-device: None
+      glance-api-version: 2
+    to:
+    - lxc:2
+  cinder-ceph:
+    charm: cs:xenial/cinder-ceph
+    num_units: 0
+  glance:
+    charm: cs:xenial/glance
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+    to:
+    - lxc:3
+  keystone:
+    charm: cs:xenial/keystone
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+      admin-password: openstack
+    to:
+    - lxc:4
+  mongodb:
+# An official Xenial mongodb charm version does not exist as of
+# 2016 Aug 25.  Use WIP/dev version instead.
+#    charm: cs:xenial/mongodb
+    charm: cs:~james-page/xenial/mongodb
+    num_units: 1
+    options:
+      source: ppa:ubuntu-s390x-community/mongodb
+    to:
+    - lxc:5
+  mysql:
+    charm: cs:xenial/percona-cluster
+    num_units: 1
+    options:
+      max-connections: 20000
+    to:
+    - lxc:1
+  neutron-api:
+    charm: cs:xenial/neutron-api
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+      neutron-security-groups: true
+      flat-network-providers: physnet1
+    to:
+    - lxc:1
+  neutron-gateway:
+    charm: cs:xenial/neutron-gateway
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+      bridge-mappings: physnet1:br-ex
+      data-port: br-ex:encc003
+    to:
+    - '1'
+  neutron-openvswitch:
+    charm: cs:xenial/neutron-openvswitch
+    num_units: 0
+  nova-cloud-controller:
+    charm: cs:xenial/nova-cloud-controller
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+      network-manager: Neutron
+    to:
+    - lxc:5
+  nova-compute:
+    charm: cs:xenial/nova-compute
+    num_units: 4
+    options:
+      openstack-origin: cloud:xenial-newton
+      enable-live-migration: true
+      enable-resize: true
+      migration-auth-type: ssh
+      cpu-mode: none
+    to:
+    - '2'
+    - '3'
+    - '4'
+    - '5'
+  ntp:
+    charm: cs:xenial/ntp
+    num_units: 0
+  openstack-dashboard:
+    charm: cs:xenial/openstack-dashboard
+    num_units: 1
+    options:
+      openstack-origin: cloud:xenial-newton
+    to:
+    - lxc:4
+  rabbitmq-server:
+    charm: cs:xenial/rabbitmq-server
+    num_units: 1
+    to:
+    - lxc:1
+  swift-proxy:
+      charm: cs:xenial/swift-proxy
```

Comment:
I believe swift-proxy and swift-storage-z? need num_unit definitions for juju 2 'juju deploy' - while this is probably not relevant now, might be worth adding in JIC